packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/alb/application-target-group.ts

```diff
@@ -330,8 +330,9 @@ export class ApplicationTargetGroup extends TargetGroupBase implements IApplicat
 
     if (props) {
       if (props.slowStart !== undefined) {
```

Comment:
Wondering if we need to handle where `slowStart` is not specified/undefined, to explicity set `slow_start.duration_seconds` attribute as default `0`.  since I saw in the issue reported, it was mentioned if it is not specified, it deploys successfully but Target Group resource still has it enabled(if previously enabled).

Reply:
hmm.. I think if they want it disabled they should have to set it explicitly ?  My concern is if we set to `0`, it will cause changes with stacks that are already deployed.

Reply:
Yeah seems valid. This looks good to me.

Reply:
awesome, thanks!